theurgy.lic

```diff
@@ -109,19 +117,51 @@ def prayer_bead
   put_in_container('carving knife')
   get_from_container('shaper')
   while checkright == 'block'
-    fput("shape block to #{$settings.aspect}")
+    fput("shape block to #{$settings.immortal_aspect}")
 	pause 1
     waitrt?
   end
   put_in_container('shaper')
   get_from_container('prayer chain')
-  fput ("put #{$settings.aspect} bead on prayer chain")
+  fput ("put #{$settings.immortal_aspect} bead on prayer chain")
   put_in_container('prayer chain')
-end    
+end
+
+def meditate_bead
+  get_from_container('prayer chain')
+  beads = DRC.bput("look at my chain", /^Strung on to the prayer bead chain you see .*\./, 'There are currently no beads on it.')
+  bead_count = DRC.list_to_nouns(beads.match(/^Strung on to the prayer bead chain you see (.*)\./).to_a[1])
+  if bead_count.count >= 10
+    fput 'kneel'
+    fput 'meditate my prayer chain'
+    pause 92
```

Comment:
will likely change this pause and rearrange some things in theurgy so that this isn't wasted time.